easybuild/easyconfigs/a/arrow-R/arrow-R-14.0.0.2-foss-2023a-R-4.3.2.eb

```diff
@@ -20,7 +20,7 @@ checksums = ['7138a52d66f1b94ec31c25e8929d6f92b1640df852a10817600a82ab68ba8ab7']
 dependencies = [
     ('R', '4.3.2'),
     ('R-bundle-CRAN', '2023.12'),
-    ('Arrow', '14.0.1'),
+    ('Arrow', '14.0.1'),  # arrow-R x.y.z[.N] only works with Arrow x.y.z
```

Comment:
@bedroge Doesn't this imply that this `arrow-R` easyconfig should be removed?

Reply:
There is definitely no good use for it anymore, but I wasn't sure what the policy is for removing easyconfigs. But since it sounds like it's fine to remove it, I've done that in https://github.com/easybuilders/easybuild-easyconfigs/pull/20324/commits/ed616365aa5c904b0807e33188cd994e1b4dffcf.

Reply:
If there's a good reason to remove it (like there is here), we should, even if it comes with fallout.

It's pretty limited here, since `arrow-R` is not a common dependency at all.